hooks/usePoolsHandler.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react"
+
+export const usePoolsHandler = () => {
+  const [poolData, setPoolData] = useState<{ asset: string; totalSupply: string; supplyApy: string; totalBorrowed: string; borrowApyVariable: string; borrowApyStable: string }[]>()
```

Comment:
separate this into a type, outside of `usePoolsHandler` something like so:
```typescript
type PoolType = {
    asset: string
    . . .
}
```

Reply:
Done